.github/workflows/tests.yml

```
@@ -41,40 +41,20 @@ jobs:
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:
           php-version: ${{ matrix.php }}
-          extensions: ${{ env.extensions }}
           tools: composer, pecl
           coverage: xdebug
 
       - uses: "ramsey/composer-install@v3"
         with:
-          composer-options: "${{ matrix.composer-options }}"
+          composer-options: "--ignore-platform-req=ext-grpc ${{ matrix.composer-options }}"
```

actionlint (1.7.10)

[error] property "composer-options" is not defined in object type {dependencies: string; php: number}

(expression)